_posts/2020-05-30-postgres-loader-0.1.0/index.md

```diff
@@ -0,0 +1,104 @@
+---
+layout: post
+title: Snowplow Postgres Loader 0.1.0 released
+title-short: Snowplow Postgres Loader 0.1.0
+tags: [postgres, shred, relational databases, storage]
+author: Anton
+category: Releases
+permalink: /blog/2020/05/30/snowplow-postgres-loader-0.1.0/
+---
+
+For last several years we were getting requests from OSS community to add PostgreSQL support to Snowplow.
+Today we're thrilled to announce the initial release of Snowplow Postgres Loader.
```

Comment:
Do we need to set some expectations early about what this should and shouldn't be used for at this point of time? I.e., you'd be brave to use this on a high volume production pipeline.

Reply:
> I.e., you'd be brave to use this on a high volume production pipeline.
I wouldn't say "brave". I'd say "experienced in Postgres administration". The Loader itself is just a dumb yet performant app that can be scaled using KCL and it can be inserting huge volumes, given enough resources. Postgres itself will be a bottleneck and user need to be aware of that in any case. Even if inserting will be fast enough - chances are that the DB will be super slow at querying. And these are good and bad of on-premise solutions.